content/tutorials/tags.json

```diff
@@ -0,0 +1,150 @@
+{
+  "Type": [
+    {
+      "value": "text",
+      "label": "Text Tutorial",
+      "description": "Tutorial in written format",
+      "color": "#010067"
+    },
+    {
+      "value": "video",
+      "label": "Video Tutorial",
+      "description": "Tutorial in video format",
+      "color": "#010067"
+    }
+  ],
+  "Topic": [
+    {
+      "value": "getting-started",
+      "label": "Getting Started",
+      "description": "Easy Tutorials for getting started",
+      "color": "#9e008e"
+    },
```

Comment:
Would it make sense to add how-to and tutorial here?
As we will need that quere for https://github.com/iota-community/iota-wiki/issues/583